applications/FluidDynamicsApplication/custom_processes/two_fluid_navier_stokes_fractional_convection_process.h

```diff
@@ -0,0 +1,646 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Uxue Chasco
+//                   
+//                   
+//
+
+#pragma once
+
+// System includes
+#include <string>
+#include <iostream>
+#include <algorithm>
+
+// External includes
+
+// Project includes
+#include "fluid_dynamics_application_variables.h"
+#include "custom_elements/two_fluid_navier_stokes_fractional_convection.h"
+#include "geometries/geometry_data.h"
+#include "solving_strategies/schemes/residualbased_incrementalupdate_static_scheme.h"
+#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h"
+#include "processes/find_nodal_h_process.h"
+#include "solving_strategies/convergencecriterias/displacement_criteria.h"
+#include "solving_strategies/strategies/residualbased_newton_raphson_strategy.h"
+
+namespace Kratos
+{
+    ///@name Kratos Globals
+    ///@{
+
+    ///@}
+    ///@name Type Definitions
+    ///@{
+
+    ///@}
+    ///@name  Enum's
+    ///@{
+
+    ///@}
+    ///@name  Functions
+    ///@{
+
+    ///@}
+    ///@name Kratos Classes
+    ///@{
+
+    /// Short class definition.
+    /**Takes a model part full of SIMPLICIAL ELEMENTS (triangles and tetras) and convects the fractional velocity variable 
+     * on the top of it as the first part of the fractinal splitting of two fluid navier stokes approach. 
+     */
+    template <unsigned int TDim, class TSparseSpace, class TDenseSpace, class TLinearSolver>
+    class TwoFluidNavierStokesFractionalConvectionProcess
+        : public Process
+    {
+    private:
+        ///@name Type Definitions
+
+        ///@}
+
+    public:
+        ///@name Type Definitions
+        ///@{
+
+        typedef ImplicitSolvingStrategy<TSparseSpace, TDenseSpace, TLinearSolver> SolvingStrategyType;
+        typedef typename BuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver>::Pointer BuilderAndSolverPointerType;
```

Comment:
```suggestion
        using SolvingStrategyType = ImplicitSolvingStrategy<TSparseSpace, TDenseSpace, TLinearSolver>;
        using BuilderAndSolverPointerType = typename typename BuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver>::Pointer;
```
Do it also for the rest of new types.